generated/nidcpower/nidcpower/session.py

```diff
@@ -4756,6 +4988,94 @@ def _parse_channel_count(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return int(number_of_channels_ctype.value)
 
+    @ivi_synchronized
+    def perform_lcr_open_compensation(self, additional_frequencies=None):
+        r'''perform_lcr_open_compensation
+
+        Generates open compensation data for LCR measurements based on a default set of test frequencies and, optionally, additional frequencies you can specify.
+
+        You must physically configure an open LCR circuit to use this method to generate valid open compensation data.
+
+        When you call this method:
+
+        -  The open compensation data is written to the onboard storage of the instrument. Onboard storage can contain only the most recent set of data.
+        -  Most NI-DCPower properties in the session are reset to their default values. Rewrite the values of any properties you want to maintain.
+
+        To apply the open compensation data you generate with this method to your LCR measurements, set the lcr_open_compensation_enabled property to **TRUE**.
+
+        Corrections for frequencies other than the default frequencies or any additional frequencies you specify are interpolated.
+
```

Comment:
I see this snippet of documentation in C function reference missing here: 
![image](https://user-images.githubusercontent.com/48041236/168179343-a26b5ea2-361e-4d39-81ce-f0d28bbba19f.png)

Why the inconsistency?

Reply:
I've added this snippet for both `perform_lcr_open_compensation()` and `perform_lcr_short_compensation()` in my latest commit